architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -12,37 +12,37 @@ Utilising the `SeriesID` was considered to show the users, but one of the long-t
 
 Discussions were also had about using the transferring body/code to identify a consignment, but was decided against due to potential for these to change.
 
-The TDR team decided that it would be helpful to generate a separate reference for users to use to refer to consignments using a base25 alphabet.
-With the introduction of the base25, the TDR backend will also be able to use either the UUID or consignment reference to look up consignments, but users will exclusively see the consignment reference within TDR pages (though TDR **URLs** will still show the consignment UUID within, since these will be used by the TDR backend)
+The TDR team decided that it would be helpful to generate a separate reference for users to refer to consignments using a base25 alphabet.
+With the introduction of the Base25, the TDR backend will also be able to use either the UUID or consignment reference to look up consignments, but users will exclusively see the consignment reference within TDR pages (though TDR **URLs** will still show the consignment UUID within, since these will be used by the TDR backend)
+
+Base25 was specifically chosen due to its ability to represent numeric identifiers through alpha-numeric codes, this is especially pertinent when it comes to longer numeric identifiers, as base25 can represent this in a more succinct and easy to read manner.
```

Comment:
I think this section should maybe still stay under the section explaining the format decision as b25 hasn't been mentioned yet. It sounds slightly like we're only doing b25 rather than the full consignment reference format